packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,144 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/master/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+---
+
+## Design
+
+Simple accordion group
+
+```
+<fast-accordion>
+    <fast-accordion-item>
+        <span slot="heading">Panel one</span>
+        <span slot="icon">^</span>
```

Comment:
I don't see an `icon` slot in the existing fast templates or specs. So this is a new api. What is it for?
Remember slots are for nimble users to add customization, what would this slots be customizing that we would want to allow customization for?

Reply:
Sorry, I pulled the `slot="icon"` part from this section in [fast's accordion spec](https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion/accordion.spec.md#implementation). Not sure why they had `icon` as a slot since they don't list it [here](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion#css-parts-1), I should have caught that!

I'll use one of the examples from the fast component explorer instead and rename the tag names to `nimble-accordion` / `nimble-accordion-item` 👍